packages/e2e-tests/helpers/utils.ts

```diff
@@ -197,12 +197,18 @@ export async function locatorTextToNumber(locator: Locator): Promise<number | nu
   return trimmed ? parseInt(trimmed) : null;
 }
 
-export async function resetTestUser(email: string) {
+export async function resetTestUser(email: string, testScope?: string) {
```

Comment:
We should probably require `testScope` since we know that calling this method _without_ a scope can break other running tests.